main.css

```diff
@@ -0,0 +1,634 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Baloo Paaji 2", cursive;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  font-family: "Baloo Paaji 2", cursive;
+  background-color: rgb(88, 88, 88);
+}
+
+nav a {
+  margin: 0;
+  padding: 0;
+  width: 250px;
+  height: 50px;
+}
+
+nav a img {
+  width: 250px;
+  height: 150px;
+  margin: -12% 2%;
+}
+
+nav ul {
+  width: 100%;
+  list-style: none;
+  display: flex;
+  margin-top: 2.2%;
+}
+
+nav ul li {
+  margin: 0 1%;
+}
+
+nav ul li a {
+  text-decoration: none;
+  font-size: 1.3em;
+  color: white;
+  font-weight: bold;
+  padding: 10px;
+}
+
+.article-body a {
+  text-decoration: none;
+  color: white;
+}
+
+.article-footer a {
+  color: white;
+}
```

Comment:
This selector has the same styles of selectors on line 84 and line 507. Don't repeat yourself, in order to minimize code you can group selectors with the same style definition.